Makefile

```diff
@@ -101,7 +101,7 @@ cache :
 	@python bin/get_workshop_info.py -v -t \
 	    -i $(CONFIG_DIR)/workshop_urls.yml \
 	    -o ./_workshop_cache.yml
-	@python bin/make-dashboard.py ./git-token.txt ./_dashboard_cache.yml
+	@python bin/make-dashboard.py -t ./git-token.txt -o ./_dashboard_cache.yml
```

Comment:
I would like to replace this with

~~~
@python bin/make-dashboard.py -t ./git-token.txt -o ./_dashboard_cache.yml || \
    python bin/make-dashboard.py -o ./_dashboard_cache.yml
~~~

If you have `git-token.txt` it will work as previous but if you does not it will prompt for your username and password.

I want to avoid doing this with

~~~
diff --git a/Makefile b/Makefile
index c4b5577..55e5c7d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+.PHONY: workshops
+
 # Base URL and installation directory for installed version on server.
 INSTALL_URL = http://software-carpentry.org
 INSTALL_DIR = $(HOME)/sites/software-carpentry.org
@@ -6,6 +8,8 @@ INSTALL_DIR = $(HOME)/sites/software-carpentry.org
 DEV_URL = http://dev.software-carpentry.org
 DEV_DIR = $(HOME)/sites/dev.software-carpentry.org
 
+CACHE_STEPS = workshops dashboard
+
 #-------------------------------------------------------------------------------
 
 # Source files in root directory.
@@ -96,11 +100,15 @@ archive :
 	    --archive $(CONFIG_DIR)/workshops_saved.yml
 
 ## cache        : collect workshop information from GitHub and store in local cache.
-cache :
+cache : ${CACHE_STEPS}
+
+workshops :
 	cp $(CONFIG_DIR)/workshops_saved.yml ./_workshop_cache.yml
 	@python bin/get_workshop_info.py -v -t \
 	    -i $(CONFIG_DIR)/workshop_urls.yml \
 	    -o ./_workshop_cache.yml
+
+dashboard : git-token.txt
 	@python bin/make-dashboard.py -t ./git-token.txt -o ./_dashboard_cache.yml
 
 ## biblio       : make HTML and PDF of bibliography.
diff --git a/README.md b/README.md
index be0b82d..00ecedf 100644
--- a/README.md
+++ b/README.md
@@ -52,8 +52,13 @@ We use Jekyll because it's what [GitHub](http://github.com/) uses;
 we use Python because most of our volunteers speak it.
 
 *   Type `make` to see a list of all available commands.
-*   Type `make cache` to fetch information about upcoming workshops.
-    This must be done before building the web site.
+*   Type `make cache` to fetch information about upcoming workshops
+    and issues at Software Carpentry's repositories at GitHub.
+
+    If you do not want to fetch issues informations, you can use
+    `make cache CACHE_STEPS=workshops` instead of `make cache`.
+
+    **This must be done before building the web site**.
 *   Type `make site` to build everything in `_site` for testing.
     (Depending on your machine, this takes about 10-15 seconds.)
 *   Type `make clean` before re-doing `make site`,
~~~

but

~~~
python ./bin/preprocess.py -c ./config -o /home/raniere/software-carpentry/site/_site -s /home/raniere/software-
carpentry/site/_site
dashboard cache file "./_dashboard_cache.yml" does not exist.
~~~